data_model/tempcron.py

```diff
@@ -0,0 +1,6 @@
+from crontab import CronTab
+
+cron = CronTab()
+
+job = cron.new(command='last_temp.py')
```

Comment:
Please try to call a function instead of a file. Try to compare with https://github.com/VTK-Revue/revue-flask-server/blob/dev/revue/scripts/__init__.py.